internal/report/source.go

```diff
@@ -917,8 +918,19 @@ func openSourceFile(path, searchPath, trim string) (*os.File, error) {
 		f, err := os.Open(path)
 		return f, err
 	}
+	possibleBases := filepath.SplitList(searchPath)
+	if gopath := os.Getenv("GOPATH"); gopath != "" {
```

Comment:
An alternative would be to somehow make sure that the SourcePath field in options is set correctly earlier?

And if it is the case that all/most Go users are using "go tool pprof" then perhaps we can just arrange for the Go invocation code to pass the right source path settings? Or are you worried about "pprof" invocations that are not via "go tool pprof"?